quora-service/src/main/java/com/upgrad/quora/service/business/AuthenticationService.java

```diff
@@ -42,11 +43,11 @@ public UserAuthTokenEntity signin(final String username, final String password)
         if (userEntity == null) {
             throw new AuthenticationFailedException("ATH-001", "This username does not exist");
         }
+        // Commented encryption as the encrypted password with salt is not matching with the user password.
+        //final String encryptedPassword = PasswordCryptographyProvider.encrypt(password, userEntity.getSalt());
 
-        final String encryptedPassword = PasswordCryptographyProvider.encrypt(password, userEntity.getSalt());
-
-        if (encryptedPassword.equals(userEntity.getPassword())) {
-            JwtTokenProvider jwtTokenProvider = new JwtTokenProvider(encryptedPassword);
```

Comment:
What is the need for this change

Reply:
I'm not able to get the encryptedPassword (generated using salt) compared with the user password. Commented the code to test the Questions endpoints. I have to fix this issue. Will work on the code.